docs/configuration.md

```diff
@@ -115,30 +115,41 @@ _e.g._
 }
 ```
 
-The `package.json#modular.type` can be `"root"`, `"app"`, `"view"` or
-`"package"`.
+The `package.json#modular.type` can be `"root"`, `"app"`, `"view"`,
+`"esm-view"`, `"source"`, `"template"` or `"package"`. Read more about Modular
+types in [this explainer](/docs/package-types).
 
 ### `"root"`
 
 This type identifies the root of the project.
 
-### `"view"`
-
-This type identifies modules that export a single React component as their
-default export. `modular` makes these modules available via a dynamically
-generated view map with `modular-views.macro`. Read more about Views in
-[this explainer](/docs/concepts/views.md).
-
 ### `"app"`
 
-This type identifies a standalone application that can be started or built.
+This type identifies a standalone Single Page Application (SPA) that can be
+started locally or built.
 
 ### `"esm-view"`
 
-This type identifies an ESM view that can be started in standalone mode, built
-and imported dynamically by an host application.
+This type identifies an ESM application that can be started in standalone mode,
+built and imported dynamically by another `esm-view`.
 
 ### `"package"`
 
-This type identifies a regular package (e.g. a library that can be used by other
-`"view"` or `"app"` modules).
+This type identifies a library that is built for consumption by other types and
+can be published to a registry like npm out of the box.
+
+### `"view"`
+
+This type identifies a module like `"package"`, that by convention exports a
```

Comment:
1. Agreed, changed it to "This type identifies a `"package"` that by convention exports a single React
component"
2. Tricky. You can't really enforce that `view`s export React components without analysing the code and applying heuristics. Re: deprecation - we hope, with modular Modular, that the distinction between `view` and `package` becomes zero - i.e., you can `modular-start` a package and get a run time error if the package doesn't export something renderable. The emphasis on `view`s is something that the original Modular versions (1 and 2) had, but it's not really so central in future Modular.
3. See below comment on `add.md` + it's hard to have a good "why" for `view`s, apart from "use a view if your package exports a React component and you want to preview it locally".

Reply:
To be honest, thinking about this, I don't really see the need for _another_ description of types in the configuration page, since we already describe there in package-types and in add. I'll remove the list and leave the link.